Data/Sys/GameSettings/RT7.ini

```diff
@@ -0,0 +1,4 @@
+# RT7E69, RT7F69, RT7P69 - Tiger Woods PGA Tour 07
+
+[Video_Settings]
+SafeTextureCacheColorSamples = 512
```

Comment:
According to my notes, JMC said that this needs `0`, not `512`. (JMC, feel free to override this)